README.md

```diff
@@ -54,6 +59,30 @@ corresponding `has_` method instead (e.g., `has_position()`):
     if not snapshot.has_mass():
         snapshot.mass = [2.,2.,10.]
 
+## Topology
+
+The particle topology is stored in `Topology` derived classes. `Topology` derived
+classes are `Bonds`, `Angles`, `Dihedrals`, and `Impropers`.
```

Comment:
```suggestion
The topology (bond information) can be stored in `Bonds`, `Angles`, `Dihedrals`,
and `Impropers` objects. All these objects function similarly, differing only in the
number of particles that are included in a connection (2 for a bond, 3 for an angle,
4 for a dihedral or improper). Each connection has an associated `id` and `typeid`.

```py
bonds = Bonds(N=3, num_types=2)
angles = Angles(N=2, num_types=1)
```

Reply:
There should be a trailing triple backtick here, but I can't figure out how to escape it inside the suggestion backticks.